features/admin/deactivate_supplier_user.feature

```diff
@@ -0,0 +1,26 @@
+@deactivate-supplier-user
+Feature: Deactivate a supplier's contributor
+
+Scenario Outline: Correct users can deactivate a supplier's contributor
+  Given I am logged in as the production <role> user
+  # TODO get here from /admin
+  And I am on the /admin/suppliers?supplier_name_prefix=DM+Functional+Test+Supplier page
+  And I click the summary table 'Users' link for 'DM Functional Test Supplier'
+  And I click the summary table 'Deactivate' button for 'DM Functional Test Supplier User 1'
+  And I click the summary table 'Activate' button for 'DM Functional Test Supplier User 1'
```

Comment:
And then a check that they have been re-activated.

Reply:
As above